bibliopixel/project/types/defaults.py

```diff
@@ -1,11 +1,13 @@
-from . import channel_order, color, colors, duration, gamma, ledtype
+from . import (
+    channel_order, color, colors, duration, gamma, ledtype, spi_interface)
 
 FIELD_TYPES = {
     'c_order': channel_order,
     'color': color,
     'colors': colors,
     'duration': duration,
     'gamma': gamma,
+    'interface': spi_interface,
```

Comment:
In this use case, interface is entirely ambiguous and I'm only comfortable with this if it's renamed `spi_interface` because `interface` might mean something else somewhere else.